components/checkbox/Checkbox.tsx

```diff
@@ -65,7 +65,7 @@ const InternalCheckbox: React.ForwardRefRenderFunction<HTMLInputElement, Checkbo
   const checkboxGroup = React.useContext(GroupContext);
   const { isFormItemInput } = useContext(FormItemInputContext);
   const contextDisabled = useContext(DisabledContext);
-  const mergedDisabled = disabled || checkboxGroup?.disabled || contextDisabled;
+  const mergedDisabled = disabled ?? checkboxGroup?.disabled ?? contextDisabled;
```

Comment:
```suggestion
  const mergedDisabled = disabled ?? checkboxGroup?.disabled || contextDisabled;
```
group 的应该还是原样，不应该有在 disabled group 里还能用的 checkbox